packages/vue-components/src/PageNavButton.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <overlay
+    v-if="showPageNav"
+    :type="'pageNav'"
```

Comment:
could do
```suggestion
    type="pageNav"
```

also for site nav, and `:fragment`

Reply:
Thanks! Updated :)